packages/react-components/react-field/src/components/Field/useFieldStyles.styles.ts

```diff
@@ -58,6 +58,7 @@ const useLabelStyles = makeStyles({
 
   horizontal: {
     marginRight: tokens.spacingHorizontalM,
+    lineHeight: tokens.lineHeightBase500,
```

Comment:
We can't change the line height, as that will affect the second+ line of the label text if it wraps.

Reply:
let me try with padding